core/src/main/scala/chisel3/internal/MonoConnect.scala

```diff
@@ -6,9 +6,9 @@ import chisel3._
 import chisel3.experimental.{Analog, BaseModule, SourceInfo}
 import chisel3.internal.containsProbe
 import chisel3.internal.Builder.pushCommand
-import chisel3.internal.firrtl.{Connect, Converter, DefInvalid}
+import chisel3.internal.firrtl.{Connect, Converter, DefInvalid, PropAssign}
 import chisel3.experimental.dataview.{isView, reify, reifyToAggregate}
-import chisel3.properties.Property
+import chisel3.properties.{Class, Property}
```

Comment:
I don't know if this is the place to nit, but `Class` is a perhaps undesirable name, even if it is descriptive, just for its meaning as Reflection of a java Class.

Reply:
It's worth thinking about but it's a separate discussion from this PR. Lets discuss changing it offline.